src/main/markdown/doc/latest/polymer-tutorial/create.md

```diff
@@ -33,12 +33,14 @@ For the **TodoList** project, we'll need to run `webAppCreator` with the followi
 
       _**Tip**: If you include the GWT SDK folder in your PATH environment variable, you won't have to specify the full path._
 
+      You may have to modify the `pom.xml` before you can run the application. Add `<type>pom</type>` to the gwt dependency, otherwise you will encounter an error. 
 
 2. Run the application in [SuperDevMode](articles/superdevmode.html).
 
      To check that the project was created correctly start the new app in SuperDevMode.
 
         $ cd TodoListApp
+        $ mvn war:exploded
```

Comment:
It could be better to do all in one line, so the user never forgets to copy resources:
    
     $ mvn war:exploded gwt:run